MLInference.py

```diff
@@ -129,9 +142,20 @@ def load_model(model_spec_file):
   model_info_dict['outputlist'] = output_list
   model_dir = model_info_dict.get('modeldirectory')
   signature = model_info_dict.get('signature')
-  imported_and_infer = get_imported_and_infer(model_dir, signature)
-  imported = imported_and_infer[0]
-  infer = imported_and_infer[1]
+
+  load_model_type = model_info_dict.get('loadmodeltype')
```

Comment:
So I think using labels like loadmodeltype is fine. But I am seeing we are starting to have a lot of those labels. It is very easy for someone to mistype the string. Can we store these keys in a table and access them using something like predefine macros?

Reply:
done, make all "key" used by macros